app/models/produce_item.rb

```diff
@@ -8,4 +8,8 @@ class ProduceItem < ApplicationRecord
 
   validates :name, presence: true
   validates :category, presence: true
+
+  def name_with_whitespace
```

Comment:
We'll use this to show the produce name to users, correct? If that's the case, we should use a more business-like method name, like `readable_name`. WDYT?